apps/protocol-frontend-e2e/src/global.d.ts

```diff
@@ -13,7 +13,12 @@ declare namespace Cypress {
   interface Chainable<Subject> {
     interceptGQL(httpMethod: Method, operationNames: string[]): void;
   }
-
+  /**
+     * Intercepts with Graphql
+  */
+     interface Chainable<Subject> {
+      mintContribution(query: string): void;
+    }
```

Comment:
Hmm looks like your prettier isn't running on this. Try running it on this file to style the code correctly